client/src/custom-types/message.d.ts

```diff
@@ -130,6 +130,8 @@ export interface CollaborationDetails {
 
 export interface MessageCustom extends CoreMessage {
   readonly messageType: typeof CUSTOM_MESSAGE
+    /** Details related to the message, excluding the message type. */
```

Comment:
@lilitkarapetyan - I _think_ we can remove `messageType` from `MessageDetails`, now that we are storing the correct` messageType` in the parent `CoreMessage`.